dartagnan/src/test/java/com/dat3m/dartagnan/others/miscellaneous/AnalysisTest.java

```diff
@@ -651,7 +651,7 @@ private AliasAnalysis analyze(Program program, Alias method) throws InvalidConfi
         ProcessingManager.fromConfig(configuration).run(program);
         Context analysisContext = Context.create();
         analysisContext.register(BranchEquivalence.class, BranchEquivalence.fromConfig(program, configuration));
-        analysisContext.register(ExecutionAnalysis.class, ExecutionAnalysis.fromConfig(program, ProgressModel.FAIR, analysisContext, configuration));
+        analysisContext.register(ExecutionAnalysis.class, ExecutionAnalysis.fromConfig(program, ProgressModel.uniform(ProgressModel.FAIR), analysisContext, configuration));
```

Comment:
Why not `ProgressModel::defaultHierarchy`?

Reply:
Same as above